pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -603,8 +603,8 @@ let
 
     microcode = {
       MICROCODE       = yes;
-      MICROCODE_INTEL = yes;
-      MICROCODE_AMD   = yes;
+      MICROCODE_INTEL = whenOlder "6.6" yes; # Removed in 6.6-RC1
```

Comment:
Not sure we need these comments — it's already pretty clear from the code, and none of the other `whenOlder`s are annotated like this.